tests/e2e/system_test/test_full_cluster_health.py

```diff
@@ -0,0 +1,306 @@
+"""
+Test to verify cluster health/stability when it's full (85%)
+"""
+
+import logging
+import time
+import pytest
+
+from ocs_ci.ocs.cluster import CephCluster, get_percent_used_capacity
+from ocs_ci.ocs import constants, benchmark_operator, ocp, defaults
+from ocs_ci.ocs.perftests import PASTest
+from ocs_ci.helpers.performance_lib import run_command
+from ocs_ci.utility import templating
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.disruptive_operations import osd_node_reboot
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.framework.pytest_customization.marks import tier3
+from ocs_ci.helpers import sanity_helpers
+
+logger = logging.getLogger(__name__)
+
+
+class TestFullClusterHealth(PASTest):
+    """
+    Test Cluster health when storage is ~85%
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, request, nodes):
+        """
+        Setting up test parameters
+        """
+
+        def teardown():
+            logger.info("cleanup the environment")
+            self.operator.cleanup()
+            logger.info(
+                "Going to sleep for 5 Minute, for background cleanup to complete"
+            )
+            time.sleep(300)
```

Comment:
please examine if maybe we can poll the cleanup process, to be sure it is completed and also to reduce the time we wait if it takes less than 5 minutes

Reply:
Removed